src/pages/home/profile/Profile.tsx

```diff
@@ -52,7 +52,7 @@ const Profile: FunctionComponent = () => {
   const windowWidth = useWindowDimensions()?.width;
   const { data: roles = [] } = useGetRolesQuery();
   const { data: genres = [] } = useGetGenresQuery();
-  const genreOptions = useExtractProperty(genres, "name");
+  const genreOptions = useExtractProperty(genres, "name", false);
```

Comment:
Not caused by this change but why is this needed, in addition I see there is validation for genre as well? This isn't a user selectable option and is obtained from the user's profile...

https://github.com/projectNEWM/newm-artist-portal/blob/a0ce3b73f08e119133b180458ed0ccc190a32ec7/src/pages/home/profile/Profile.tsx#L142